core/server/services/auth/authorize.js

```diff
@@ -38,7 +38,15 @@ const authorize = {
     },
 
     authorizeAdminAPI: [session.ensureUser],
-    // used by API v2 endpoints
+    authorizeContentApi(req, res, next) {
```

Comment:
@kevinansfield I wanted to double check with you about this

Reply:
Removing `req.user` checking from content api authorization - is that correct, or incorrect?